gutenberg.php

```diff
@@ -27,9 +27,12 @@
  * @since 0.1.0
  */
 function the_gutenberg_project() {
+	global $post_type, $post_type_object;
+	$post_type_object = get_post_type_object( $post_type );
```

Comment:
I don't have much WP experience but it seems like `$post_type_object ` might be null. I don't know if that will be caught in another place so we might provide fallback:
```
$screen_reader_text = ! empty( $post_type_object->labels->edit_item ) ? $post_type_object->labels->edit_item : __( 'Default text' );
```

It would make sense to double check with someone else. /cc @pento

Reply:
We can assume that `$post_type_object` is set by the time `the_gutenberg_project()` is called.

For WordPress 4.9+:
- For new posts, `$post_type` is sanity checked at the start of `post-new.php`, then `$post_type_object` is set. Later, the `replace_editor` filter is run, which calls `gutenberg_init()`, which may call `the_gutenberg_project()`.
- For existing posts, `$post_type_object` is set at the start of `post.php`, which bails before the `replace_editor` filter is run if `$post_type_object` is empty.

For <WordPress 4.9, `gutenberg_intercept_edit_post()` and `gutenberg_intercept_post_new()` simulate the same behaviour.

So, we don't need to include the `$post_type_object = get_post_type_object( $post_type );` line, we can just access the `$post_type_object` global immediately.